packages/node/src/lib/novu.ts

```diff
@@ -40,8 +41,25 @@ export class Novu extends EventEmitter {
   readonly organizations: Organizations;
   readonly workflowOverrides: WorkflowOverrides;
 
-  constructor(apiKey: string, config?: INovuConfiguration) {
+  constructor(config?: INovuConfiguration);
+  constructor(apiKey: string, config?: INovuConfiguration);
+  constructor(...args: any) {
     super();
+
+    let apiKey: string | undefined;
+    let config: INovuConfiguration | undefined;
+
+    if (arguments.length === 2) {
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** 
‏
```suggestion
    const haveApiKeyParam = arguments.length === 2;
    if (haveApiKeyParam) {
```

Just for easier readability.